proxy/pkg/flagset/flagset.go

```diff
@@ -43,6 +46,14 @@ func HealthWithConfig(cfg *config.Config) []cli.Flag {
 	}
 }
 
+func mustUserConfigDir() string {
+	dir, err := os.UserConfigDir()
+	if err != nil {
+		panic(err)
+	}
+	return path.Join(dir, "proxy")
```

Comment:
could we add a folder `ocis` to the path? 
-> eg. $HOME/.config/ocis/proxy/server.crt

This would add a more precise oCIS context and helps you keeping your .config folder clean ;-) This also applies to all other artifacts generated in this PR

Reply:
good idea. Also easier to cleanup

Reply:
```
tree /Users/aunger/Library/Application\ Support/ocis
/Users/aunger/Library/Application\ Support/ocis
├── ldap
│   ├── ldap.crt
│   └── ldap.key
└── proxy
    ├── server.crt
```

better organized